ballerina/tests/record-mapping-test.bal

```diff
@@ -235,3 +235,115 @@ function queryTypedRecordWithoutFieldsClosed() returns error? {
         test:assertFail("Error expected");
     }
 }
+
+annotation ColumnConfig TestColumn on record field;
+
+public type Album record {|
+    @Column{
```

Comment:
```suggestion
    @Column {
```

Reply:
check other places

Reply:
Fixed in [c55c66d](https://github.com/ballerina-platform/module-ballerina-sql/pull/509/commits/c55c66d1a7bc85e22b29ffd1f34a442aec7c63ca)